packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -167,6 +173,63 @@ const sharedStyles = StyleSheet.create({
     },
 });
 
+const action = semanticColor.action.outlined.progressive;
+
+// NOTE: This color is only used here.
+const pink = "#fa50ae";
```

Comment:
It'll be nice when we have the full color palette so we won't have to hardcode one-off colors!